composer.lock

```diff
@@ -177,37 +177,33 @@
         },
         {
             "name": "mockery/mockery",
-            "version": "1.6.2",
+            "version": "1.6.4",
```

Comment:
I think projects using `wp_mock` can still install older versions of Mockery, as long as they match the `"mockery/mockery": "^1.6",` restriction. Not a bad thing in this case, but I wanted to point that out.

I don't know when is the right time to bump minimum requirements in a public repo like this one. Perhaps we need a specific reason for doing so.

In our private packages, we could depend on mockery directly too, to set the minimums that we want.